src/test/resources/examples/githubApi/models/kotlinx/Event.kt

```diff
@@ -0,0 +1,18 @@
+package examples.githubApi.models
+
+import javax.validation.constraints.NotNull
+import kotlin.Any
+import kotlin.String
+import kotlin.collections.Map
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+public data class Event(
+  @SerialName("entity_id")
+  @get:NotNull
+  public val entityId: String,
+  @SerialName("data")
+  @get:NotNull
+  public val `data`: Map<String, Any?>,
```

Comment:
wrt your pre-review comment around [@Contextual](https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-core/kotlinx.serialization/-contextual/), are we being inconsistent by generating this, but omitting additional properties? I'm wondering would it be better to error out rather than produce code that:
a.) Doesn't compile
b.) Results in data loss (is it possible that kotlinx will quietly drop additional properties?)

Reply:
Lets be consistent and error out in case untyped objects or additional properties are found in the schema.

Supporting untyped objects with `@Contextual` is totally viable, but we need to figure out how to build it in a user friendly way.

Support for additionalProperties is being tracked in [this issue](https://github.com/Kotlin/kotlinx.serialization/issues/1978) by @pschichtel.